cost/aws/burstable_instance_cloudwatch_credit_utilization/aws_burstable_instance_cloudwatch_credit_utilization.pt

```diff
@@ -547,50 +553,76 @@ script "js_check_merged_metrics", type: "javascript" do
   EOS
 end
 
-
 ###############################################################################
-# Policy
+# Escalations
 ###############################################################################
 
-policy "pol_utilization" do
-  validate $ds_check_merged_metrics do
-    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} rows containing AWS instance CloudWatch utilization data"
-    detail_template <<-EOS
-# AWS Instance Utilization
-| Region | instanceID | Instance Type | Hostname | Private DNS Name | CPU Maximum % | CPU Average % | CPU Credit Balance Minimum | CPU Surplus Credit Balance Minimum | CPU Surplus Credit Charged Maximum | Max Earnable Credits | Resize Instance To |
-| ------ | ---------- | ------------- | -------- | ---------------- | ------------- | ------------- | -------------------------- | ---------------------------------- | ---------------------------------- | -------------------- |------------------- |
-{{ range data -}}
-| {{.region}} | {{.instanceId}} | {{.instanceType}} | {{.hostname}} |  {{.privateDnsName }} | {{ .cpu_maximum }} | {{ .cpu_average }} | {{if eq .cloudwatch_cpu_credit_balance_min "10000"}}N/A{{else}}{{.cloudwatch_cpu_credit_balance_min}}{{end}} | {{if eq .cloudwatch_cpu_surplus_credit_balance_min "10000"}}N/A{{else}}{{.cloudwatch_cpu_surplus_credit_balance_min}}{{end}} | {{if eq .cloudwatch_cpu_surplus_credits_charged_max "10000"}}N/A{{else}}{{.cloudwatch_cpu_surplus_credits_charged_max}}{{end}} | {{.instance_type_info.burst_info.max_earnable_credits}} | {{.next_instance_size}} |
-{{ end -}}
-___
-###### Policy Applied in Account: {{ rs_project_name }} (Account ID: {{ rs_project_id }}) within Org: {{ rs_org_name }} (Org ID: {{ rs_org_id }})
-EOS
-    escalate $email
-    escalate $resize_instance
-    check eq(size(data), 0)
-  end
+escalation "esc_email" do
+  automatic true
+  label "Send Mail"
+  description "Sends incident email"
+  email $param_email
 end
 
+escalation "esc_resize_instance" do
+  automatic contains($param_automatic_action, "Re-Size Instances")
+  label "Resize Instances"
```

Comment:
We should be consistent.  Resize is spelled multiple ways, Re-size and Resize.  I think it should just be Resize.

Reply:
modified the policy